docs/standards.md

```diff
@@ -22,17 +22,37 @@ Additionally, here are some common variable suffixes that are recommended:
 `_f`    for a pointer to a function
 `_ctx`  for context data i.e. data that is passed into a function
 
-### Functions
+#### Functions
 Exported function names are prefixed, and lowercase snake case.
+
+The prefix used is based on the name of the dynamic library and the functionality it provides.
+```
+  DynamicLibraryLoaderHelper -> DLLH_
```

Comment:
Could this be misleading for circumstances where it's not actually a DLL? Like if it's a `dylib`?